### PR TITLE
MH: create account

### PR DESCRIPTION
**Pull request by mhedloff**

### CODE REVIEW

File: clientApp/src/app/components/create-user/create-user.component.html [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+            <mat-label>Password</mat-label>
+            <input matInput type="password" formControlName="password">
+          </mat-form-field>
+          <mat-form-field>
+            <mat-label>Confirm password</mat-label>
+            <input matInput type="password" formControlName="confirmPassword">
+          </mat-form-field>
+        </div>
+      </mat-card-content>
+      {{ this.createUserForm.errors | json }}
```
Comment (HerrKuless) [inline clientApp/src/app/components/create-user/create-user.component.html]: create a proper error notification, a json is not readable for a simple user and it is also not very pretty

File: clientApp/src/app/services/auth/authentication.service.ts [hunk 2]
```diff
@@ -23,7 +23,7 @@ export class AuthenticationService {
 	public isAuthenticated(): boolean {
     const decodedToken = AuthenticationService.decodeToken();
     if (decodedToken) {
-      const expirationDate = fromUnixTime(decodedToken.exp);
+      const expirationDate = decodedToken.exp * 1000;
```
Comment (HerrKuless) [inline clientApp/src/app/services/auth/authentication.service.ts]: create a const

File: clientApp/src/app/services/auth/authentication.service.ts [hunk 3]
```diff
@@ -49,4 +49,10 @@ export class AuthenticationService {
           : null;
       }));
 	}
+
+  public createAccount(username: string, password: string): Observable<boolean> {
+    return this.http.post('https://127.0.0.1:4848/user', {
```
Comment (HerrKuless) [inline clientApp/src/app/services/auth/authentication.service.ts]: create a const

File: clientApp/src/app/components/create-user/create-user.component.html (hunk 1 above)
Comment (mhedloff) [reply clientApp/src/app/components/create-user/create-user.component.html]: sure, i overlooked it :D

**Pull request merged**